force-app/main/default/objects/Education_History__c/fields/Status__c.field-meta.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Status__c</fullName>
+    <description>The status of the Education History. For example, In Progress or Completed.</description>
```

Comment:
We don't have an In Progress picklist value. I've reached out to docs on this one to get a final text example here, but we will need to change this.

Reply:
Update on this. We should change `In Progress` to `Enrolled`

Reply:
Fixed